net_handler/udp_conn.c

```diff
@@ -94,6 +94,7 @@ static void* send_device_data(void* args) {
                     }
                     param__init(param);
                     param->name = device_info->params[j].name;
+                    param->readonly = device_info->params[j].write == false;
```

Comment:
is this a change that you made? or is this a merge conflict with the base branch

Reply:
This is also from PR #185